pkg/aws/eni/routing/routing.go

```diff
@@ -42,6 +43,12 @@ var (
 // mtu: The ENI device MTU.
 // masq: Whether masquerading is enabled.
 func (info *RoutingInfo) Configure(ip net.IP, mtu int, masq bool) error {
+	if ip.To4() == nil {
+		log.WithFields(logrus.Fields{
+			"IP": ip,
+		}).Warning("Unable to Configure, the endpoint address is not an IPv4 address in ENI mode")
```

Comment:
```suggestion
		log.WithFields(logrus.Fields{
			"endpointIP": ip,
		}).Warning("Unable to configure rules and routes because IP is not an IPv4 address")
```

Reply:
Done.